src/Main.java

```diff
@@ -1,5 +1,24 @@
+import lesson6tasks.Circle;
+import lesson6tasks.Dog;
+import lesson6tasks.Point;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        Rectangle rectangleFirst = new Rectangle(12,23);
+        Circle circleFirst = new Circle(12,3.14);
+        Dog dogFirst = new Dog("Retriever",2,"Doggy");
```

Comment:
чому для створення кола передаєьбся 3.14? для всіх колів, це буде однкавою константою. 
```suggestion
        Circle circleFirst = new Circle(12,3.14);
        Dog dogFirst = new Dog("Retriever",2,"Doggy");
```